overlayGoogleRemote/html/index.html

```diff
@@ -31,12 +31,14 @@ <h1 class="display-3">CamOverlay Remote Control</h1>
             <form class="myForm">
               <div class="form-group">
                 <label for="accessToken">Google Sheet Endpoint Adress</label><br>
+                <div style="overflow-wrap: break-word;">
                 <small><i>
-                  Step 1: Publish sheet to the web: File > Publish to the web > Publish <br>
-                  Step 2: Share your sheet with public settings: Anyone on the internet with this link can view.<br>
-                  Step 3: Copy only sheet ID from sheet URL:  e.g. <br>https://docs.google.com/spreadsheets/d/<b>abcde12345BpVoRMQv3J_6yFpSWuSwtRf5QnyqT_qyOI</b>/edit#gid=0<br> (the bold part)
+                  Step 1: Create the Google Sheer and click "File" - "Publish to web"Publish sheet to the web: File > Publish to the web > Publish <br>
```

Comment:
Google Sheer je neco co neznam nebo typo?

Reply:
nezapomel jsi tam i stary text?

Reply:
to je nový text od Pavla 
Sheer opravím

Reply:
zmatla me ta absence mezery za uvozovkama..

Reply:
tak mi to Pavel poslal, nevím co to má znamenat, kdyžtak se snad ozve

Reply:
ale mezeru tam teda hodím